base/array.jl

```diff
@@ -2,6 +2,38 @@
 
 ## array.jl: Dense arrays
 
+"""
+    Array{T}(dims)
+    Vector{T}(m)
+    Matrix{T}(m, n)
+
+`Array{T}(dims)` constructs an uninitialized dense array with element type `T`. `dims` may
+be a tuple or a series of integer arguments, each giving the size of the corresponding
+dimension. `Vector{T}(m)` and `Matrix{T}(m, n)` are stricly equivalent aliases to create one-
```

Comment:
this phrasong makes it sound like Vector and Matrix are equivalent to each other